tools/flaky-test-reporter/slack_notification.go

```diff
@@ -137,6 +136,21 @@ func createSlackMessageForRepo(rd *RepoData, flakyIssuesMap map[string][]*flakyI
 	return message
 }
 
+// getSlackChannel gets the channel(s) to send messages to for a given job configuration
+func getSlackChannel(rd *RepoData) ([]slackChannel, bool) {
+	jobMap, ok := slackChannelsMap[rd.Config.Repo]
+	if !ok {
+		log.Printf("cannot find Slack channel mapping for repo '%s', skipping Slack notification", rd.Config.Repo)
+		return nil, ok
+	}
+	channels, ok := jobMap[rd.Config.Name]
+	if !ok {
+		log.Printf("cannot find Slack channel for job '%s' in repo '%s', skipping Slack notification", rd.Config.Name, rd.Config.Repo)
+		return nil, ok
+	}
+	return channels, ok
```

Comment:
seems like this function always return either `nil, false` or `channels, true`, but not the other combinations, is `ok` really needed to be returned?